assets/js/components/UI/Tabs/StickyHeader.jsx

```diff
@@ -8,7 +8,7 @@ const UITabsStickyHeader = ({ title, children, ...restProps }) => {
       position: "-webkit-sticky",
       position: "sticky",
       top: headerHeight,
-      zIndex: 1110,
+      zIndex: 39,
```

Comment:
Ha, I just actually changed this as well.  Can you change it to `10`?  I've found when dealing with multiple `z-index` elements in an app, it's generally easier to go in increments of 10.  ie. 10, 20, 30, 40, 50, etc.